packages/flutter/lib/src/material/bottom_navigation.dart

```diff
@@ -0,0 +1,536 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+import 'dart:collection' show Queue;
+
+import 'package:flutter/widgets.dart';
+import 'package:vector_math/vector_math_64.dart' show Vector3;
+
+import 'colors.dart';
+import 'icon.dart';
+import 'icon_theme.dart';
+import 'icon_theme_data.dart';
+import 'ink_well.dart';
+import 'material.dart';
+import 'theme.dart';
+
+const double kBottomNavigationHeight = 60.0;
```

Comment:
Call this kBottomNavigationBarHeight for consistency. It should go in constants.dart.